composer.json

```diff
@@ -50,9 +50,9 @@
         "php-di/php-di": "^6.0.0",
         "phpmailer/phpmailer": "^6.1",
         "psr/log": "~1.0",
-        "symfony/console": "~2.6.0",
-        "symfony/event-dispatcher": "~2.6.0",
-        "symfony/monolog-bridge": "~2.6.0",
+        "symfony/console": "~5.3.0",
```

Comment:
Can we upgrade to 5.4 ? I don't think this will imply more works but this will for sure secure a little bit more the support of these dependencies 😉

Reply:
5.4 wasn't released when this PR was created, but with security support until 30 Nov 2025 we might want to switch to 5.4 now.
@sgiehl ?